core-bundle/tests/EventListener/DataContainer/PageRoutingListenerTest.php

```diff
@@ -484,4 +546,23 @@ public function testSkipsSimilarPageIfUrlDoesNotMatch(): void
 
         $this->assertSame('', $listener->generateRouteConflicts($dc));
     }
+
+    /**
+     * @return PageRoute&MockObject
+     */
+    private function mockPageRoute(string $path, array $requirements = []): PageRoute
```

Comment:
The `$requirements` argument is never used. Are the tests perhaps incomplete?

<img width="771" alt="" src="https://user-images.githubusercontent.com/1192057/149158165-504c2454-e34a-44b3-a65f-c641751f0ba2.png">

Reply:
Good catch. The tests were actually complete but because I duplicated the method instead of using it 😂 fixed in 3540ce501714be444c7397759c84b83af35e30c6